README.md

```diff
@@ -189,6 +198,19 @@ end
 order.certificate # => PEM-formatted certificate
 ```
 
+### Ordering an alternative certificate
+
+Let's Encrypt is transitioning to use a new, self-signed intermediate certificate. Starting September 29, 2020 new certificates will be signed by their own intermediate. To ease the transition on clients Let's Encrypt will continue signing an alternative version of the certificate using the old, cross-signed intermediate until September 29, 2021. In order to utilize an alternative certificate the `Order#certificate` method accepts a `preferred_chain` keyword argument. 
+For example, to download the cross-signed certificate after September 29, 2020, call `Order#certificate` as follows:
+ 
+```ruby
+order.certificate(preferred_chain: 'IdenTrust') || order.certificate
+``` 
+
+Note: if the specified peferred chain doesn't match an existing alternative certificate it will return `nil`.
```

Comment:
Typo

```suggestion
Note: if the specified preferred chain doesn't match an existing alternative certificate it will return `nil`.
```

Reply:
I think the current implementation is actually going to raise an error, and is depending on the consumer of the gem to catch `Acme::Client::Error::PreferredChainNotMatched`. Correct me if I'm wrong.

See https://github.com/unixcharles/acme-client/pull/190/files#diff-e843384610a1a0cb8c21582bbed52188R151